code/game/objects/structures/hybrisa_props.dm

```diff
@@ -0,0 +1,2359 @@
+	icon_state = "xenocellemptyon"
+	density = TRUE
+
+/obj/structure/prop/urban/xenobiology/small/offempty
+	name = "specimen containment cell"
+	desc = "It's turned off and empty."
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics.dmi'
+	icon_state = "xenocellemptyoff"
+	density = TRUE
+
+/obj/structure/prop/urban/xenobiology/small/larva
+	name = "specimen containment cell"
+	desc = "There is something worm-like inside..."
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics.dmi'
+	icon_state = "xenocelllarva"
+	density = TRUE
+
+/obj/structure/prop/urban/xenobiology/small/egg
+	name = "specimen containment cell"
+	desc = "There is, what looks like some sort of egg inside..."
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics.dmi'
+	icon_state = "xenocellegg"
+	density = TRUE
+
+/obj/structure/prop/urban/xenobiology/small/hugger
+	name = "specimen containment cell"
+	desc = "There's something spider-like inside..."
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics.dmi'
+	icon_state = "xenocellhugger"
+	density = TRUE
+
+/obj/structure/prop/urban/xenobiology/small/cracked1
+	name = "specimen containment cell"
+	desc = "Looks like something broke it...from the inside."
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics.dmi'
+	icon_state = "xenocellcrackedempty"
+	density = TRUE
+
+/obj/structure/prop/urban/xenobiology/small/cracked2
+	name = "specimen containment cell"
+	desc = "Looks like something broke it...from the inside."
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics.dmi'
+	icon_state = "xenocellcrackedempty2"
+	density = TRUE
+
+/obj/structure/prop/urban/xenobiology/small/crackedegg
+	name = "specimen containment cell"
+	desc = "Looks like something broke it, there's a giant empty egg inside."
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics.dmi'
+	icon_state = "xenocellcrackedegg"
+	density = TRUE
+
+/obj/structure/prop/urban/xenobiology/big
+	name = "specimen containment cell"
+	desc = "A giant tube with a hulking monstrosity inside, is this thing alive?"
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics2.dmi'
+	icon_state = "bigqueencryo1"
+
+/obj/structure/prop/urban/xenobiology/big/bigleft
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics2.dmi'
+	icon_state = "bigqueencryo1"
+	layer = ABOVE_MOB_LAYER
+
+/obj/structure/prop/urban/xenobiology/big/bigright
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics2.dmi'
+	icon_state = "bigqueencryo2"
+	layer = ABOVE_MOB_LAYER
+
+/obj/structure/prop/urban/xenobiology/big/bigbottomleft
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics2.dmi'
+	icon_state = "bigqueencryo3"
+	density = TRUE
+	layer = ABOVE_MOB_LAYER
+
+/obj/structure/prop/urban/xenobiology/big/bigbottomright
+	icon = 'icons/obj/structures/prop/urban/urbanxenocryogenics2.dmi'
+	icon_state = "bigqueencryo4"
+	density = TRUE
+	layer = ABOVE_MOB_LAYER
```

Comment:
Uh, does this actually need to be 5 different objects instead of 1 big one?

Reply:
I'm not sure why this was done, if it was all together it would be 32 * 128 though.

Reply:
Its not a huge deal, but we have multi tile structures without issue

Reply:
Please tell me if this is something that needs addressed before TM

Reply:
Nah